### PR TITLE
Adding fitsverify

### PR DESCRIPTION
**Pull request by jhunkeler**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -12,4 +12,4 @@ pr:
 jobs:
   - template: ./.azure-pipelines/azure-pipelines-linux.yml
   - template: ./.azure-pipelines/azure-pipelines-osx.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
+  #- template: ./.azure-pipelines/azure-pipelines-win.yml
```
Comment (chrisburr) [inline azure-pipelines.yml:15]: You shouldn't modify files outside of the `recipes/PACKAGE_NAME/` directory

File: recipes/fitsverify/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://heasarc.gsfc.nasa.gov/docs/software/ftools/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e4f8d34414ad441414386c81f2d241c800025cd1ad21c269d061484e617cef8
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cfitsio
```
Comment (chrisburr) [inline recipes/fitsverify/meta.yaml:18]: This belongs in `host` not `run`
```suggestion
```

File: recipes/fitsverify/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://heasarc.gsfc.nasa.gov/docs/software/ftools/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3e4f8d34414ad441414386c81f2d241c800025cd1ad21c269d061484e617cef8
+
+build:
+  number: 0
```
Comment (chrisburr) [inline recipes/fitsverify/meta.yaml:13]: If you don't want to support windows (and it's not a `noarch` package) then this is the right thing to do:
```suggestion
  number: 0
  skip: true  # [win]
```

File: recipes/fitsverify/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - bzip2
+    - libcurl
+    - cfitsio
+  run:
+    - cfitsio
+
+test:
+  commands:
+    - ${PREFIX}/bin/fitsverify -h
```
Comment (chrisburr) [inline recipes/fitsverify/meta.yaml:29]: This should be on `$PATH`:
```suggestion
    - fitsverify -h
```

**Comment by jhunkeler** [comment]
@chrisburr Thanks for the edits. The reason I added cfitsio to `run` not `host` was because I was running into the "error" I'm receiving here. 

```
   INFO (fitsverify,bin/fitsverify): Needed DSO lib/libcfitsio.so.8 found in conda-forge::cfitsio-3.470-hb418390_7
  ERROR (fitsverify,bin/fitsverify): Needed DSO lib/libcurl.so.4 found in ['libcurl']
  ERROR (fitsverify,bin/fitsverify): .. but ['libcurl'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (fitsverify,bin/fitsverify): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-h77966d4_13
  ERROR (fitsverify,bin/fitsverify): Needed DSO lib/libbz2.so.1.0 found in ['bzip2']
  ERROR (fitsverify,bin/fitsverify): .. but ['bzip2'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (fitsverify,bin/fitsverify): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-h77966d4_13
```

`cfitsio` requires the these dependencies, not `fitsverify`. Why do these need to be redefined as dependencies when they're already dependencies of `cfitsio`?

**Comment by chrisburr** [comment]
If you're linking to `libcurl` and `bzip2` (which `bin/fitsverify` is doing) you need to track the ABI of those dependencies and they're needed in `host`.

**Comment by chrisburr** [comment]
The "less likely" message should really be the other way around, it's almost always that the dependency is missing from the host requirements.

File: recipes/fitsverify/build.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+export PKG_CONFIG_PATH="$BUILD_PREFIX/lib/pkgconfig:$PREFIX/lib/pkgconfig:$PKG_CONFIG_PATH"
```
Comment (chrisburr) [inline recipes/fitsverify/build.sh:3]: Do you need this?
```suggestion
```

**Pull request merged**